### DISCUSSION TITLE
Localize image title

### DISCUSSION THREAD

**Discussion by kiyoshi-satoo** [wagtail/wagtail-localize#730] [Q&A] [answered]
Are there any ways to translate the title of an image when using Image Chooser form?

**Comment by zerolab** [comment, marked as answer]
Hey @kiyoshi-satoo, I do not have the capacity to test this week, but having a custom image model with `TranslatableMixin` may work. Now, Wagtail doesn't support that out of the box, so you may be stuck with using duplicate images if you want to translate things 🤔

**Reply by kiyoshi-satoo** [reply]
Hi @zerolab. Yes, I thought I would have to create a custom model. But unfortunately there are a lot of pictures loaded. I'll have to find a way out of it :)  Thanks for the reply anyway
